src/lib/utils/index.ts

```
@@ -29,6 +29,48 @@
+        "state",
+        "post_login_redirect_url",
+        "authUrlParams",
+        "redirect_url"
+      ].includes(key)
+    ) {
+      delete paramsObject.authUrlParams[key];
+    }
+  });
+
+  if (Object.keys(paramsObject.authUrlParams).length === 0) {
+    delete paramsObject.authUrlParams;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)